framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/ClusteredWorkbenchKieServerPersistentScenarioImpl.java

```diff
@@ -168,4 +174,52 @@ public SsoDeployment getSsoDeployment() {
     public GitProvider getGitProvider() {
         return gitProvider;
     }
+
+    @Override
+    protected String overridesVersionTag() {
```

Comment:
Wouldn't it have more sense to be able to control version to be deployed directly from test?

Reply:
Initially, I set it directly from the test, however the OperatorConstants is not available from there, so either:

1. we configure this from pom.xml as it is now
2. create a new argument for general purpose (not only operator) that will only be used for operators (I don't really like this).

What do you think?

Reply:
the current approach sounds good.